### DISCUSSION TITLE
v6 - A standard way

### DISCUSSION THREAD

**Discussion by Acrontum-Carmichael** [acr-lfr/generate-it#101] [Ideas]
Before breaking apart the tool into multiple components that can be used in an orchestration with a conductor aka a cli tool we need to standardise the object the core generates and passes to the template renderer.

**Comment by p-mcgowan** [comment]
I guess to start, we should figure out which parta of thw specs (paths, components, etc) we need, and what would be an easily processable data structure.

If need be, it might also help to pass around the original json spec, as well as (maybe) the un-dereffed schema
